INPUT.md

```diff
@@ -0,0 +1,373 @@
+
+The `math_function` object defines how loads change over time. 
+
+```JSON
+{
+
+    "math_function": [
+        {
+            "id": 0,
+            "type": "Linear",
+            "xvalues": [0.0, 1.0, 10.0],
+            "fxvalues": [0.0, 1.0, 1.0]
+        }
+    ]
+
+}
+```
+
+* `"id"` corresponds the applied boundary loading above. 
+* `"type"` is currently limited to linear functions. 
+* `"xvalues"` is the model time. 
+* `"fxvalues"` is the relative weight of the force or traction magnitude defined in the loading conditions. 
+
+## Analysis
+
+The `analysis` object defines the type of analysis, time step, and total number of steps.
+
+```JSON
+{
+
+    "analysis": {
+        "type": "MPMExplicit2D",
+        "mpm_scheme": "usf",
+        "velocity_update": true,
+        "dt": 1.0e-5,
+        "nsteps": 1001,
+        "locate_particles": true,
+        "nload_balance_steps": 100,
+        "uuid": "output-name-000",
+        "resume": {
+            "resume": false,
+            "uuid": "output-name-000",
+            "step": 100
+        },
+        "damping": {
+            "type": "Cundall",
+            "damping_factor": 0.01
+        }
+    }
+
+}
+```
+
+* `"velocity_update"` flag is set `true` to use nodal velocity to update particle velocity. 
+* `"dt"` is the time step. 
+* `"nsteps"` is the total number of steps. 
+* `"locate_particles"` flag is set `false` to allow simulation to continue if material points leave the mesh domain. 
+* `"nload_balance_steps"` is **optional** to redefine the dynamic load balancing; default is every 1000 steps. 
+* `"uuid"` is the analysis name. 
+
+### Type
+
+Supported types:
+
+| Analysis      | Description     |
+|---------------|-----------------|
+| MPMExplicit2D | explicit 2D MPM |
+| MPMExplicit3D | explicit 3D MPM |
+
+### Scheme
+
+Support schemes:
+
+| Schemes | Description                 |
+|---------|-----------------------------|
+| usf     | update stress first         |
+| usl     | update stress last          |
+| musl    | modified update stress last |
+
+### Resume
+
+Resume is optional.
+
+### Damping
+
+Damping is optional. Currently only Cundall damping can be specified. 
+
+## Post Processing
+
+The `post_processing` object defines what output files are generated. 
+
+```JSON
+{
+
+    "post_processing":{
+        "output_steps": 100,
+        "path": "results/",
+        "vtk": [ ],
+        "vtk_statevars": [
+            {
+                "phase_id": 0,
+                "statevars": [ ]
+            }
+        ]
+    }
+
+}
+```
+
+* `"output_steps"` is the number of steps between generated files. 
+* `"path"` is where output files will be saved; path will be generated if it does not already exist. 
+
+### VTK
+
+Available VTK files include `stresses`, `strains`, `displacements`, and `velocities`.
+
+### VTK State Variables
+
+State variables are generated based on their phase ID (e.g., solid is `0`, liquid is `1`). Available state variables are material dependent. 
+
+
```

Comment:
@bodhinandach I'm not even sure if this is true since I mostly use the HDF5 output. More details might make sense here.

Reply:
Yes, it is still used. I am using that all the time

Reply:
Yes, I am using this all the time. Please keep it this way.

Reply:
That wasn't my questions... I'm wondering what are the options for VTK? (i.e., are there other outputs available?)